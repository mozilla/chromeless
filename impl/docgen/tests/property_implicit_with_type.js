/**
 * A module to test implicit property documentation,
 * where the developer provides only a description
 */

/**
 * a property which exposes a name to you
 * @type string
 */
exports.name = "Lloyd Hilaiel";
