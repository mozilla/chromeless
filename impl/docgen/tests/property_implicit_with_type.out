{
  "desc": "A module to test implicit property documentation,\nwhere the developer provides only a description", 
  "filename": "tests/property_implicit_with_type.js", 
  "module": "property_implicit_with_type", 
  "properties": [
    {
      "desc": "a property which exposes a name to you", 
      "name": "name", 
      "type": "string"
    }
  ]
}
